src/tours.tsx

```diff
@@ -92,6 +94,57 @@ export const tours: Record<TourId, TourStep[]> = {
       ),
     },
   ],
+  // Launched after importing data without connecting a micro:bit.
+  [TourId.DataSamplesPagePostImport]: [
+    {
+      title: <FormattedMessage id="tour-dataSamples-imported-title" />,
+      content: (
+        <FormattedMessageStepContent id="tour-dataSamples-imported-content" />
+      ),
+      modalSize: "xl",
+    },
+    {
+      selector: classSelector(tourElClassname.dataSamplesActionCard),
+      title: <FormattedMessage id="actions-label" />,
+      content: (
+        <FormattedMessageStepContent id="tour-dataSamples-imported-actions-content" />
+      ),
+    },
+    {
+      selector: classSelector(tourElClassname.recordDataSamplesCard),
+      title: <FormattedMessage id="content.data.data" />,
+      content: (
+        <FormattedMessageStepContent id="tour-dataSamples-imported-recordings-content" />
+      ),
+    },
+    {
+      selector: classSelector(tourElClassname.addActionButton),
+      title: (
+        <FormattedMessage id="tour-dataSamples-imported-addActions-title" />
+      ),
+      content: (
+        <FormattedMessageStepContent id="tour-dataSamples-imported-addActions-content" />
+      ),
+    },
+    {
+      selector: classSelector(tourElClassname.connectButton),
+      title: (
+        <FormattedMessage id="tour-dataSamples-imported-collectData-title" />
+      ),
+      content: (
+        <FormattedMessageStepContent id="tour-dataSamples-imported-collectData-content" />
+      ),
+    },
+    {
+      selector: classSelector(tourElClassname.trainModelButton),
+      title: <FormattedMessage id="menu.trainer.trainModelButton" />,
+      content: (
+        <FormattedMessageStepContent id="tour-collectData-trainModel-content" />
+      ),
+    },
+    // TODO: Add tour step for triggering data samples page tour
+    // so that the user can see tour steps of things they may have missed
```

Comment:
Adding a tour after connecting micro:bit was considered so that other content could also be introduced e.g. recording instructions, live graph. But depending on which page/stage the user connects a micro:bit, the content would probably need to be different. As a more generic response, perhaps we add a step at the end of the tour